docs/preview/02-Features/powershell/azure-devops.md

```diff
@@ -109,6 +137,24 @@ PS> Set-AzDevOpsArmOutputsToPipelineVariables -ArmOutputsEnvironmentVariableName
 # ##vso[task.setvariable variable=my-variable]my-value
 ```
 
+This function is intended to be used from an Azure DevOps pipeline.
+> Note that when you are using a Linux agent, you need to pass environment variables as these are not available. To be able to use the ArmOutputs environment variable, it must be explicitly added to the environment-variables.
```

Comment:
```suggestion

> Note that when you are using a Linux agent, you need to pass other environment variables that you want to use as well, because these are not available. To be able to use the `ArmOutputs` environment variable, it must be explicitly added to the environment-variables.
```